swagger/specification/applications/resource-manager/Applications.Core/preview/2023-10-01-preview/openapi.json

```diff
@@ -2745,6 +2745,10 @@
             "type": "object"
           }
         },
+        "recipeConfig": {
```

Comment:
Do you know why recipe related properties are getting added to Application properties? I understand it's not directly related to this PR, but weird to see it here.

Reply:
I think it should be `ApplicationProperties` instead of `EnvironmentProperties`https://github.com/radius-project/radius/blob/0a1a8680dace317f8bd25bdde2b503f895766d2d/typespec/Applications.Core/applications.tsp#L145
Maybe a typo while creating this file. Will create a fix for this in different PR.
Good Catch!!

Reply:
@vishwahiremat let's create a tracking issue so that this gets picked up during next sprint planning.

Reply:
I have created a PR for this here :https://github.com/radius-project/radius/pull/7182

Reply:
Just noticed https://github.com/radius-project/radius/pull/7182/files. Thanks for following up on this quickly.